library/common/jni_interface.cc

```diff
@@ -158,6 +180,22 @@ static void jvm_on_complete(void* context) {
 }
 
 // Utility functions
+static void jni_delete_global_ref(void* context) {
```

Comment:
Would it be better to move these utility functions to a different file?

Reply:
Yeah, we can start cleaning this up in subsequent PRs